index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="./style.css" />
+    <title>Personal Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <nav>
+        <a href="#" class="logo">My logo</a>
+        <img src="./img/menu.svg" alt="" />
+      </nav>
+    </header>
+
+    <section class="content">

```

Comment:
[optional]

I noticed you have 2 wrappers ('content' and 'subcontent').
 You can consider removing one.

Please take into consideration that the next step is to build the work cards. Those will have to be in a completely separate `section` container. If you consider building the next milestone as a 'subcontent' class, kindly, replace the `section` tag with `main` and the 'subontent' `div` with a `section` tag.